.gitignore

```diff
@@ -19,3 +19,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+pnpm-lock.yaml
```

Comment:
Since it is a public package, it is better to lock the dependencies using npm, instead of pnpm

Reply:
ok, makes sense